testlib/helper.go

```diff
@@ -169,7 +169,7 @@ func (h *MainHelper) PreloadMigrations() {
 	handle := h.SQLStore.GetMaster()
 	_, err = handle.Exec(string(buf))
 	if err != nil {
-		mlog.Error("Error preloading migrations. Check if you have &multiStatements=true in your DSN if you are using MySQL. Or perhaps the schema changed? If yes, then update the warmup files accordingly.")
+		mlog.Critical("Error preloading migrations. Check if you have &multiStatements=true in your DSN if you are using MySQL. Or perhaps the schema changed? If yes, then update the warmup files accordingly.")
```

Comment:
We can also do a errors.Wrap(msg, err) and feed that to panic. I wrote this code. I just wanted this custom error message to appear along with the original error. We don't really need two separate logs. Single panic with the combined error message is sufficient.